tests/phpunit/elementor/includes/template-library/test-manager-cloud.php

```diff
@@ -1052,4 +1066,210 @@ public function test_get_quota_fails_without_source() {
 
 		$this->assertEquals( 'The required argument(s) "source" not specified.', $result->get_error_message() );
 	}
+
+	public function test_import_template_with_empty_path() {
+		// Act
+		$result = $this->cloud_source_mock->import_template('test.json', '');
+
+		// Assert
+		$this->assertWPError($result);
+		$this->assertEquals('file_error', $result->get_error_code());
+		$this->assertEquals('Please upload a file to import', $result->get_error_message());
+	}
+
+	public function test_import_template_with_quota_exceeded() {
+		// Arrange
+		$template_data = [
+			'content' => ['test_content'],
+			'page_settings' => ['test_settings'],
+			'title' => 'Test Template',
+			'type' => 'container'
+		];
+
+		$this->cloud_library_app_mock
+			->method('get_quota')
+			->willReturn([
+				'currentUsage' => 100,
+				'threshold' => 100
+			]);
+
+		$this->cloud_source_mock
+			->method('prepare_import_template_data')
+			->willReturn($template_data);
```

Comment:
```suggestion

		$this->cloud_source_mock
			->method(' prepare_import_template_data' )
			->willReturn( $template_data );
```
Check the whole file